Controllers.test/Controllers.test.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Please follow the same TargetFramework for all projects. I could see both net6.0 and net7.0 being used.

Reply:
Sure Vasanth. We have updated the TargetFramework for all the project.